pyproject.toml

```diff
@@ -315,7 +318,146 @@ ignore = [
   # pairwise-over-zipped (>=PY310 only)
   "RUF007",
   # mutable-class-default
-  "RUF012"
+  "RUF012",
+  # Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks
+  # "E721",
+  # Parenthesize `a and b` expressions when chaining `and` and `or` together, to make the precedence clear
+  "RUF021",
```

Comment:
To confirm, are these rules pylint rules that are currently ignore in pandas?

Reply:
hi @mroeschke, it's not. They are some left over rules while I'm trying to figure out how to run preview mode with pylint only. I pushed a new commit to make the intent clearer. 

Now in `pyproject.toml`, you can find 2 sections, `"Output of pylint-to-ruff"`, where the `pylint` config is automatically translated to `ruff` and `"Additional pylint rules"` are the extra rules we need to ignore to make the check pass. I think most of these rules are new rules that taken inspiration from `pylint`'s existing rules.